esi_leap/api/controllers/v1/utils.py

```diff
@@ -27,7 +27,9 @@ def get_offer_authorized(uuid_or_name, cdict, status_filter=None):
 
         if not status_filter or o.status == status_filter:
             try:
-                policy.authorize('esi_leap:offer:offer_admin', cdict, cdict)
+                if o.project_id != cdict['project_id']:
```

Comment:
Can you add tests that cover this being true and this being false?

Reply:
done.